### ISSUE TITLE
Change the track bar default to 6

### ISSUE THREAD

**Issue by Blackcipher101** [Blackcipher101/Spaceye#7]
The trackbar default should be setup to 6 its 7 currently

**Comment by Xx-Ashutosh-xX** [comment]
can you tell me more on this. @Blackcipher101

**Comment by Blackcipher101** [comment]
When you check the track bar it should start out at the value of 6 not 7 as of right now the trackbar basically decides the amount smoothness one wants

**Comment by Xx-Ashutosh-xX** [comment]
# If I am correct, you are talking about this...right?

![Screenshot (215)](https://user-images.githubusercontent.com/58273769/101981318-f8fa2680-3c91-11eb-9bef-741988011be6.png)

**Comment by Blackcipher101** [comment]
Yes but this value should be changed in gui and also in opencv

**Comment by Xx-Ashutosh-xX** [comment]
meaning, I have to implement something like a slider so that the user could change its values, right?

**Comment by Blackcipher101** [comment]
> meaning, I have to implement something like a slider so that the user could change its values, right?

That is already implemented I just want to change the default value it takes at the starting which is 7.

**Comment by Xx-Ashutosh-xX** [comment]
@Blackcipher101 I checked this one out apparently the ksize (amount of blur) must be an odd number. 
https://plantcv.readthedocs.io/en/v1.1/median_blur/#:~:text=Applies%20a%20median%20blur%20filter,the%20OpenCV%20function%20median%20blur.

**Comment by Blackcipher101** [comment]
Yeah its true the blur value is calculated (slider)*2+1=blur so it always odd just change the slider value

**Comment by Blackcipher101** [comment]
@Xx-Ashutosh-xX

**Comment by Xx-Ashutosh-xX** [comment]
```
[sg.Slider(range=(1,8),
         default_value=4,
         size=(20,15),
         orientation='horizontal',
         font=('Helvetica', 12),
         key="-BLUR-"),]
```
So, I have to just change the default value to 6 from 4 ?

**Comment by Blackcipher101** [comment]
yep
